release_description.md

```diff
@@ -0,0 +1,14 @@
+### The Amazon QLDB team is pleased to announce the release of v1.2.0 of the QLDB .NET driver. This release adds asynchronous support to the driver.
+
+* Add the asynchronous version of the driver.
+* Improved retry logic to now handle more types of failure.
+
+The full list of changes are included in the [change log](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/CHANGELOG.md).
+
+Here is some sample usage for creating an async driver and executing a query:
+
```

Comment:
We can add three sections:

Create asynchronous driver:
```
AmazonQLDBSessionConfig amazonQLDBSessionConfig = new AmazonQLDBSessionConfig();
string ledgerName = "quick-start";
IAsyncQldbDriver driver = AsyncQldbDriver.Builder()
    .WithQLDBSessionConfig(amazonQLDBSessionConfig)
    .WithLedger(ledgerName)
    .Build();
```


Execute a query:
```
IAsyncResult result = await driver.Execute(async txn => await txn.Execute("SELECT * from testTable"));
```

Iterate through result:
```
await foreach (IIonValue ionValue in result )
{
    Console.WriteLine(ionValue.ToPrettyString());
}
```

Reply:
Done.